website/docs/components/reveal/partials/code/component-api.md

```diff
@@ -0,0 +1,16 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="text" @required="true" @type="string">
+    Plain text string which will appear on the toggle button which controls hiding and showing of the content. If no text value is defined an error will be thrown.
+  </C.Property>
+  <C.Property @name="textWhenOpen" @type="string">
+    Plain text which displays on the toggle button while the content is displayed.
+  </C.Property>
+  <C.Property @name="isOpen" @values={{array "false" "true" }} @default="false" @type="boolean">
+    Used to control whether the content is displayed or hidden on page load.
```

Comment:
Maybe this was copied from another place, but I wonder if this is accurate? It doesn't just control on page load, but toggles the visibility when the toggle is interacted with, doesn't it? Or am I missing something? WDYT?

Reply:
The only reason I was thinking the consumer would make use of it is to change the default value to true so that the content displays on page load. I guess I should explain more fully what it does though, thanks.

Reply:
I updated the explanation to be more complete. Please let me know if you have other suggestions for improvement though, thanks.